StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -42,6 +42,7 @@ extension PaymentSheet {
         case dynamic(String)
         case UPI
         case cashApp
+        case grabPay
```

Comment:
Adding LPM's would be a bit easier if we ripped this type out, as we now have two types representing the ~same thing.